apps/Angry IP scanner/uninstall

```diff
@@ -11,7 +11,7 @@ reduceapt() { #remove unwanted lines from apt output
   grep -v "apt does not have a stable CLI interface.\|Reading package lists...\|Building dependency tree\|Reading state information...\|Need to get\|After this operation,\|Get:\|Fetched\|Selecting previously unselected package\|Preparing to unpack\|Unpacking \|Setting up \|Processing triggers for "
 }
 
-sudo apt purge -y ipscan 2>&1 | reduceapt
+"${DIRECTORY}/pkg-purge" "ipscan" "$(dirname "$0")" || error "Failed to purge Angry IP Scanner!"
```

Comment:
This line is not needed as purge-installed will do the work here
This line can be removed

Reply:
Oops, I forgot to remove it.